internal/cypress/sauce/runner.go

```diff
@@ -78,31 +86,48 @@ func (r *Runner) runSuites(fileID string) int {
 
 	// Collect results.
 	errCount := 0
+	completed := 0
+	total := len(r.Project.Suites)
+	inprogress := total
+
+	log.Info().Msgf("Suites completed: %d in progress: %d", completed, inprogress)
 	for i := 0; i < len(r.Project.Suites); i++ {
-		r := <-results
-		if r.err != nil {
-			log.Err(r.err).Str("suite", r.suiteName).Msg("Suite failed.")
+		res := <-results
+		completed++
+		inprogress--
+		logSuite(completed, inprogress, res.suiteName, res.browser, res.job.Passed)
+
+		if res.err != nil {
+			assetContent, err := r.JobReader.GetJobAssetFileContent(context.Background(), res.job.ID, resto.ConsoleLogAsset)
+			if err != nil {
+				log.Err(res.err).Str("suite", res.suiteName).Msg("Failed to get job asset.")
+			} else {
+				fmt.Println(string(assetContent))
+			}
 			errCount++
```

Comment:
at this point we wouldnt need this, if we are getting the results with the status of the suite, lets starting saving the actual value to show as exit code

Reply:
If one of the test suites not passed then we will return exit code 1. Is it okay?